flocker/node/_deploy.py

```diff
@@ -143,6 +146,16 @@ class StartApplication(object):
     :ivar NodeState node_state: The state of the node the ``Application``
         is running on.
     """
+    application = field(type=Application, mandatory=True)
+    node_state = field(type=NodeState, mandatory=True)
+
+    @property
```

Comment:
I left a comment mentioning that these properties use `start_action` essentially because of laziness on my part.  For that same reason, I'm leaving `eliot_action` as an attribute.  When we have time, someone can define an `ActionType` for each of these (at least, for each that doesn't get deleted!) and since eventually I think Eliot is planning to remove the logger argument required to instantiate an `ActionType`, it will be possible to turn these into normal attributes rather than properties (left a comment to that effect as well).